docs/02-app/02-api-reference/05-next-config-js/reactMaxHeadersLength.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: reactMaxHeadersLength
+description: The maximum length of the headers that are emitted by React and added to the response.
+---
+
+During static rendering, React can emit headers that can be added to the response. These can be used to improve performance by allowing the browser to preload resources like fonts, scripts, and stylesheets. The default value is `6000`, but you can override this value by configuring the `reactMaxHeadersLength` option in `next.config.js`:
```

Comment:
Should we also add <picture data-single-emoji=":nextjs:" title=":nextjs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/nextjs/2a85d633ae594556.png" alt=":nextjs:" align="absmiddle"></picture> <picture data-single-emoji=":docs:" title=":docs:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/docs/172e1b9b7d137b8a.png" alt=":docs:" align="absmiddle"></picture> here → https://rc.nextjs.org/docs/app/building-your-application/styling/stylesheets about manually preloading stylesheets?